### PR TITLE
Feature/ivars

### PR DESCRIPTION
**Pull request by IvarsSaudinis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="navbar navbar-dark bg-dark box-shadow">
+        <div class="container d-flex justify-content-between">
+            <a href="#" class="navbar-brand d-flex align-items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="mr-2"><path d="M23 19a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h4l2-3h6l2 3h4a2 2 0 0 1 2 2z"></path><circle cx="12" cy="13" r="4"></circle></svg>
+                <strong>Album</strong>
+                 <a href="#" class="text-white">Saite no Alus/ivars komandas</a>
```
Comment (thugc) [inline index.html:49]: Wehrs da alus thing?! no alus no party

File: album.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+:root {
+    --jumbotron-padding-y: 3rem;
```
Comment (thugc) [inline album.css:2]: W is for Wumbo

File: index.html (hunk 1 above)
Comment (thugc) [inline index.html:49]: kur ir aluscilvēks?

**Review by thugc — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (IvarsSaudinis) [reply index.html:49]: work 1st, beer 2nd

**Pull request merged**